### PR TITLE
Updated baden-wuerttemberg.de ruleset.

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/baden-wuerttemberg.de.xml [hunk 1]
```diff
@@ -98,6 +95,7 @@
 	<target host="haushalt2017.baden-wuerttemberg.de" />
 	<target host="hochwasser.baden-wuerttemberg.de" />
 	<target host="www.hochwasser.baden-wuerttemberg.de" />
+	<target host="www.hvz.baden-wuerttemberg.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/baden-wuerttemberg.de.xml]: See also https://github.com/EFForg/https-everywhere/pull/14177/files#diff-455192acc695a43efed546fd8e1eba97R11

File: src/chrome/content/rules/baden-wuerttemberg.de.xml [hunk 2]
```diff
@@ -143,10 +146,6 @@
 	<target host="www.vm.baden-wuerttemberg.de" />
 	<target host="wm.baden-wuerttemberg.de" />
 
-	<test url="http://www.beteiligungsportal.baden-wuerttemberg.de/" />
-	<test url="http://www.mvi.baden-wuerttemberg.de/" />
-	<test url="http://www.mwk.baden-wuerttemberg.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/baden-wuerttemberg.de.xml]: They need to be added as targets, see rules below.

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/baden-wuerttemberg.de.xml]: Sorry, my wrong.

File: src/chrome/content/rules/baden-wuerttemberg.de.xml [hunk 3]
```diff
@@ -123,7 +121,9 @@
 	<target host="www.mlr.baden-wuerttemberg.de" />
 	<target host="mvi.baden-wuerttemberg.de" />
 	<target host="www.mvi.baden-wuerttemberg.de" />
+	<target host="www.mvi.baden-wuerttemberg.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/baden-wuerttemberg.de.xml]: That one is duped

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**